### PR TITLE
docs: add tutorial blob-url

### PR DESCRIPTION
**Pull request by kocoten1992**
## Description

Fixes: https://github.com/google/shaka-player/issues/1481

## Checklist:

- [x] I have signed the Google CLA <https://cla.developers.google.com>
- [x] My code follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have verified my change on multiple browsers on different platforms
- [x] I have run `./build/all.py` and the build passes
- [x] I have run `./build/test.py` and all tests pass

### CODE REVIEW

**Comment by kocoten1992** [comment]
Hi @joeyparrish, this PR is ready for review

File: lib/net/http_fetch_plugin.js [hunk 1]
```diff
@@ -290,4 +290,8 @@ if (shaka.net.HttpFetchPlugin.isSupported()) {
       'https', shaka.net.HttpFetchPlugin.parse,
       shaka.net.NetworkingEngine.PluginPriority.PREFERRED,
       /* progressSupport= */ true);
+  shaka.net.NetworkingEngine.registerScheme(
```
Comment (joeyparrish) [inline lib/net/http_fetch_plugin.js:293]: Can you please move this change to a separate PR?  That way, we can cherry-pick this PR to the older release branches to document the process for them, without importing this new feature.  Thanks!

File: docs/tutorials/blob-url.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Blob URL
+
+In case you need shaka-player to load a blob url mpd:
+
+#### For version v2.x
+
+- Register blob scheme to network engine (note: you don't have to do this in v3.x)
```
Comment (joeyparrish) [inline docs/tutorials/blob-url.md:7]: This is not quite true.  You don't have to do it in v3.2+.

File: docs/tutorials/blob-url.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  `shaka.net.NetworkingEngine.registerScheme('blob', shaka.net.HttpFetchPlugin);`
+- With blobs, neither extension nor MIME type can be used to deduce which manifest parser to use.
+  But there is an easy fix is register manifestParserFactory when load player
+  `player.load: player.load(uri, null, shaka.dash.DashParser);`
```
Comment (joeyparrish) [inline docs/tutorials/blob-url.md:11]: Not sure why you have `player.load: ` before the call.  Would you please remove it?

File: docs/tutorials/blob-url.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- Relative URIs in the manifest are broken because the original manifest URI is lost.
+  This can be worked around with either absolute URIs or the use of BaseURL element in DASH.
```
Comment (joeyparrish) [inline docs/tutorials/blob-url.md:13]: These comments about relative URIs are generally applicable to all versions of Shaka Player, and to HLS as well as DASH.  Would you please move them to a separate section at the top?

File: docs/tutorials/blob-url.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+#### For version v3.x (v3.2 and up)
```
Comment (joeyparrish) [inline docs/tutorials/blob-url.md:15]: I think you mean v3.1 and below, since you're adding this code to v3.2 already.

**Review by joeyparrish — CHANGES_REQUESTED** [review]
Thanks for contributing!

**Comment by denissabramovs** [comment]
Why this is still not merged? This is pretty useful stuff. I was looking for a way to play blobs coming from websocket connection.

**Comment by theodab** [comment]
> Why this is still not merged? This is pretty useful stuff. I was looking for a way to play blobs coming from websocket connection.

The author has not yet responded to Joey's change requests.

**Comment by kocoten1992** [comment]
thanks @avelad !

**Pull request closed without merging**